WcaOnRails/app/webpacker/components/RolesTab/DelegateForm.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import { Form } from 'semantic-ui-react';
+import I18n from '../../lib/i18n';
+
+const delegateStatusOptions = [{
+  text: I18n.t('enums.user.delegate_status.trainee_delegate'),
+  value: 'trainee_delegate',
+}, {
+  text: I18n.t('enums.user.delegate_status.candidate_delegate'),
+  value: 'candidate_delegate',
+}, {
+  text: I18n.t('enums.user.delegate_status.delegate'),
+  value: 'delegate',
+}, {
+  text: I18n.t('enums.user.delegate_status.senior_delegate'),
+  value: 'senior_delegate',
+}];
```

Comment:
Can you just store a constant of `delegate_status` enum values somewhere, and then "magically" derive the i18n values when needed through `I18n.t('enums.user.delegate_status.${some_status}')`?